Makefile.toml

```diff
@@ -286,6 +286,52 @@ category = "ICU4X WASM"
 command = "rm"
 args = ["-rf", "wasmpkg"]
 
+### VALGRIND TASKS ###
+
+[tasks.valgrind]
+description = "Build ICU4X example files with default features and run through Valgrind"
```

Comment:
Suggestion: I think the valgrind command could use some documentation that explains what it is, how it works, and how to view the results. I believe `tools/benchmark/README.md` would be appropriate place to do this.

Reply:
In case it's helpful, this is what I did locally:
```
pip install gprof2dot
gprof2dot --format=callgrind --output=out.dot benchmarks/valgrind/tui.out
dot -Tpng out.dot -o graph.png
eog graph.png
```

Reply:
Added to README.md as suggested.